### PR TITLE
[ Release ] NNTrainer 0.1.1 Release

### PR DESCRIPTION
**Pull request by jijoongmoon**
NNTrainer v0.1.1 is released

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

**Review by kparichay — APPROVED** [review]
LGTM

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+nntrainer (0.1.1) UNRELEASED; urgency=medium
+
+  * 0.1.1 released
+
+ -- jijoongmoon <jijoongmoon@jijoong-DeskTop>  Wed, 23 Sep 2020 15:26:32 +0900
```
Comment (zhoonit) [inline debian/changelog:5]: Fix email?

**Review by zhoonit — APPROVED** [review]
LGTM

File: debian/changelog (hunk 1 above)
Comment (jijoongmoon) [reply debian/changelog:5]: Oh.. Thanks.

**Pull request merged**